wpiutil/src/main/native/include/wpi/StringMap.h

```diff
@@ -548,91 +495,6 @@ class StringMapKeyIterator
   std::string_view Key;
 };
 
-template <typename ValueTy>
```

Comment:
We need this comparison operator (we need to be able to fully compare the content of StringMaps).